sentence_encoder.py

```diff
@@ -52,14 +41,9 @@ def encode(self, docs: DocumentArray = [], parameters: Dict = {}, **kwargs):
             attribute get an embedding.
         :param parameters: Any additional parameters for the `encode` function.
         """
-        document_batches_generator = DocumentArray(
-            filter(
-                lambda d: d.text,
-                docs[parameters.get('access_paths', self.access_paths)],
-            )
-        ).batch(batch_size=parameters.get('batch_size', self.batch_size))
+        batch_generator = (docs[pos:pos + self.batch_size] for pos in range(0, len(docs), self.batch_size))
```

Comment:
@jupyterjazz Just curious that is the `.batch()` deprecated in v2?

Reply:
yes v2 doesn't have this functionality